common/brave_isolated_worlds.h

```diff
@@ -11,6 +11,9 @@
 enum BraveIsolatedWorldIDs {
     // Isolated world ID for Greaselion (Google Translate reserves END + 1)
     ISOLATED_WORLD_ID_GREASELION = content::ISOLATED_WORLD_ID_CONTENT_END + 2,
+#if defined(OS_ANDROID)
+    ISOLATED_WORLD_ID_COSMETIC_SCRIPTS = content::ISOLATED_WORLD_ID_MAX - 1,
```

Comment:
I think our id managing is broken now. We are using same id with chrome's.
It would be good to fix this also here? :) 
```
enum ChromeIsolatedWorldIDs {
  // Isolated world ID for Chrome Translate.
  ISOLATED_WORLD_ID_TRANSLATE = content::ISOLATED_WORLD_ID_CONTENT_END + 1,

  // Isolated world ID for internal Chrome features.
  ISOLATED_WORLD_ID_CHROME_INTERNAL,

#if defined(OS_MAC)
  // Isolated world ID for AppleScript.
  ISOLATED_WORLD_ID_APPLESCRIPT,
#endif  // defined(OS_MAC)

  // Numbers for isolated worlds for extensions are set in
  // extensions/renderer/script_injection.cc, and are are greater than or equal
  // to this number.
  ISOLATED_WORLD_ID_EXTENSIONS
};
```
https://source.chromium.org/chromium/chromium/src/+/master:chrome/common/chrome_isolated_world_ids.h

Reply:
@simonhong what do you mean what id do you propose to take? I took the max id - 1, because the latest in the enum you posted `ISOLATED_WORLD_ID_EXTENSIONS` is actually a first id in the list of unknown size for extensions.

Reply:
We took it wrong for `ISOLATED_WORLD_ID_GREASELION`, becuase it's actually a `ISOLATED_WORLD_ID_CHROME_INTERNAL` but we don't use it at all.

Reply:
Hmm, I think patching to upstream file is one method like below.
```
enum ChromeIsolatedWorldIDs {
  // Isolated world ID for Chrome Translate.
  ISOLATED_WORLD_ID_TRANSLATE = content::ISOLATED_WORLD_ID_CONTENT_END + 1,

  // Isolated world ID for internal Chrome features.
  ISOLATED_WORLD_ID_CHROME_INTERNAL,

#if defined(OS_MAC)
  // Isolated world ID for AppleScript.
  ISOLATED_WORLD_ID_APPLESCRIPT,
#endif  // defined(OS_MAC)
  BRAVE_ISOLATED_IDS,  // <====== Brave
  // Numbers for isolated worlds for extensions are set in
  // extensions/renderer/script_injection.cc, and are are greater than or equal
  // to this number.
  ISOLATED_WORLD_ID_EXTENSIONS
};
```

and in `chromium_src`, `BRAVE_ISOLATED_IDS` is defined with ours. WDYT?

Reply:
I can do that, no problem, but I don't know it's one extra patch, do we need that on this point? We can always add a patch, and every patch is making rebase longer, but in the other hand it's a small file. @bridiver wdyt?

Reply:
I think it could be conflict if we use `max-1` number. From extensions, they could use ids upto max-1.

Reply:
I'm not sure we can avoid id overlapping w/o patching.

Reply:
there are no extensions support on Android

Reply:
@SergeyZhukovsky Ah!, I forgot about it. Then I think we could `ISOLATED_WORLD_ID_MAX`.

Reply:
I'll pass this decision to @bridiver .
I think using `ISOLATED_WORLD_ID_MAX` for android seems fine now. 
However, if we want to add ids on desktop in the future, this is not extensible and we should revisit this discussion again.

Reply:
I don't think any of this is necessary including the existing greaselion id. Use `ISOLATED_WORLD_ID_CHROME_INTERNAL`